AccountingSystem/AccountingSystem/CSVHandler.h

```diff
@@ -0,0 +1,176 @@
+#pragma once
+#include <string>
+#include <functional>
+#include <vector>
+#include <cstdio>
+#include <experimental/filesystem>
+#include "IDataHandler.h"
+
+template <class T>
+class CSVHandler : public IDataHandler<T>
+{
+public:
+	 CSVHandler(std::string pathToFile, std::vector<std::string> columnsName, std::function<std::string(T)> convertToInputFormat,
```

Comment:
Let's keep them as separate functions for now (convert ones). But we might think about moving them into T class and also whether it's beneficial to declare `IDataRow` and work with interface instead of templates.

Moving functions to the object is good because it's Object-Oriented design. Also, getting rid of templates makes code a little easier. Although, with interface we would not be able to return by value and would need to use shared pointer or something, which is worse. Thus let's keep it as it is, since I'm not sure what is better